core/renderers/zelos/info.js

```diff
@@ -547,9 +553,10 @@ Blockly.zelos.RenderInfo.prototype.finalizeVerticalAlignment_ = function() {
         prevSpacer.height += this.constants_.SMALL_PADDING;
       } else if (hasNextNotch) {
         // Determine if the input row has non-shadow connected blocks.
-        var hasNonShadowConnectedBlocks = false;
-        var MIN_VERTICAL_TIGHTNESTING_HEIGHT = 40;
-        for (var j = 0, elem; (elem = row.elements[j]); j++) {
+        let hasNonShadowConnectedBlocks = false;
+        const MIN_VERTICAL_TIGHTNESTING_HEIGHT = 40;
```

Comment:
Should this be moved to a top-level declaration?

Reply:
This doesn't need increased scope, but since it's not a top-level declaration I have updated it to use `camelCase` according to the style guide: https://google.github.io/styleguide/jsguide.html#naming-local-variable-names